mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -25,12 +25,18 @@
 		<i class="icon-check"></i>
 	</h3>
 
-	<aui:form cssClass="task-form" name="task-form">
-		<aui:input cssClass="input-item" name="task" placeholder="enter-task" required="true" />
+	<portlet:resourceURL id="task-resource" var="taskListURL" />
+
+	<aui:form action="<%= taskListURL.toString() %>" cssClass="task-form" id="taskForm" method="post" name="taskForm">
+		<aui:input cssClass="input-item" id="task-input" name="task" placeholder="enter-task" required="true" />
 
 		<aui:button cssClass="add-task" name="add" type="submit" value="add" />
 	</aui:form>
 
+	<div id="<portlet:namespace />request-success"></div>
```

Comment:
@mateomustapic This comment relates to UX. I'm not sure if you had chance to discuss with @markocikos and @kresimir-coko Liferay puts messages at the top of window. I guess you guys always can google particular problem and check what is best-practice etc. Here is example where no effort is needed (so you have to replace tesxt container position) and you can gain much better usability :)